ports/abseil/vcpkg.json

```
@@ -1,14 +1,14 @@
 {
   "name": "abseil",
-  "version-string": "2020-03-03",
-  "port-version": 8,
+  "version-string": "2020-09-23",
   "description": [
     "an open-source collection designed to augment the C++ standard library.",
     "Abseil is an open-source collection of C++ library code designed to augment the C++ standard library. The Abseil library code is collected from Google's own C++ code base, has been extensively tested and used in production, and is the same code we depend on in our daily coding lives.",
     "In some cases, Abseil provides pieces missing from the C++ standard; in others, Abseil provides alternatives to the standard for special needs we've found through usage in the Google code base. We denote those cases clearly within the library code we provide you.",
     "Abseil is not meant to be a competitor to the standard library; we've just found that many of these utilities serve a purpose within our code base, and we now want to provide those resources to the C++ community as a whole."
   ],
   "homepage": "https://github.com/abseil/abseil-cpp",
+  "supports": "linux | osx | ((x64 | arm64) & windows)",
```

```suggestion
  "supports": "(x64 | arm64) & (linux | osx | windows)",
```

32 bit Linux is a thing and absl doesn't support that; note that this will only have effects for community triplets.